projects/plugins/jetpack/modules/google-fonts/jetpack-google-fonts.php

```diff
@@ -0,0 +1,48 @@
+<?php
```

Comment:
I was curious if you had a specific reason to move this to a new file instead of leaving it in `modules/google-fonts.php`? I figure on the long term, the list of fonts may be what we want to edit the most often, and the first place folks will look for it will be `modules/google-fonts.php`, the entry point? What do you think?

Reply:
Sure! Moving it to the main file is a good idea.

Reply:
Agreed, having to find your way to the required file with the actual module code seem unnecessary.